pom.xml

```diff
@@ -70,15 +70,15 @@
   </profiles>
 
   <properties>
-    <jackson.version>2.9.8</jackson.version>
+    <jackson.version>2.9.9</jackson.version>
     <skip.tests>false</skip.tests>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>18.0</version>
+      <version>27.1-jre</version>
```

Comment:
Please change this to `28.0-android`.

Reply:
update to latest andriod, lets see how jdk7 support